connector/connect/server/src/test/scala/org/apache/spark/sql/connect/service/SparkConnectStreamingQueryCacheSuite.scala

```diff
@@ -95,7 +95,9 @@ class SparkConnectStreamingQueryCacheSuite extends SparkFunSuite with MockitoSug
     // Query is returned when correct session is used
     assert(sessionMgr.getCachedQuery(queryId, runId, mockSession).contains(mockQuery))
 
-    // Stop the query.
+    // Cleanup the query and verify if stop() method has been called.
+    sessionMgr.cleanupRunningQueries(sessionHolder)
```

Comment:
This is not testing the desired functionality. It is just testing this cleanup method, not how it is used. we want the session to timeout normally and associated queries should be closed. 

Can we add a test for that? Also, we should manually verify this.